temba/contacts/models.py

```diff
@@ -188,7 +190,7 @@ def validate(cls, urn, country_code=None):
                 return False
 
         # facebook uses integer ids or temp ref ids
-        elif scheme == cls.FACEBOOK_SCHEME:
+        elif scheme in [cls.FACEBOOK_SCHEME]:
```

Comment:
@Robi9 I think we need to add some validation for Instagram URN in this method

I think adding that at https://github.com/rapidpro/rapidpro/blob/44f12b8593231ecb33fc652a5766515fa474bd2f/temba/contacts/models.py#L205 is enough.

Remember to add a test for that too

Reply:
@Robi9 why not simply add Instagram to the other all digits here and not mix that with facebook one that supports referral as well